src/sentry/tasks/post_process.py

```diff
@@ -1118,6 +1123,12 @@ def process_code_mappings(job: PostProcessJob) -> None:
             org_slug = org.slug
             next_time = timezone.now() + timedelta(hours=1)
 
+            # php automatic code mappings currently in LA
+            if "php" in event.data["platform"] and not features.has(
+                "organizations:derive-php-code-mappings", org
+            ):
+                return
+
```

Comment:
this is the only relevant change in this file, the rest are automatic linter changes